dan_layer/consensus/src/hotstuff/on_ready_to_vote_on_local_block.rs

```diff
@@ -191,8 +195,8 @@ where TConsensusSpec: ConsensusSpec
 
         // Store used for transactions that have inputs without specific versions.
         // It lives through the entire block so multiple transactions can be sequenced together in the same block
-        let tree_store = ChainScopedTreeStore::new(block.epoch(), block.shard(), tx);
-        let mut substate_store = PendingSubstateStore::new(*block.parent(), tree_store);
+        // let tree_store = ChainScopedTreeStore::new(block.epoch(), block.shard_group(), tx);
```

Comment:
I decoupled the tree calculations from the substate store since I needed to implement a per shard JMT and I wanted that logic to be separate from the substate store. Basically, I looked at it and it didn't make sense to me for it to be in there